### ISSUE TITLE
OnGestureCompleted not called in build project

### ISSUE THREAD

**Issue by Byte247** [MARUI-PlugIn/MiVRy#4]
Hello,

we are new to unity and the MiVRy plugin, but we managed to get it working for a couple of custom gestures in unity. However when we build the project MiVRy does not work for us anymore.
Specifically it looks like the  "OnGestureCompleted" function is never called.

Is there something specific we need to keep in mind to make it also work in a build project? Or should it always work when it works in the unity "preview game window"?

**Comment by Byte247** [comment]
Or it seems to work on a windows build, but not for Android and the Quest

**Comment by max-krichenbauer** [comment]
Hello,

There are instructions for building the Unity plug-in for Android in the documentation PDF:
https://github.com/MARUI-PlugIn/MiVRy/blob/master/unity/Documentation.pdf

Most importantly:
- make sure that the plugin file (`.so`) is selected for your Android platform in Unity Inspector.
- make sure that you actually "download" the gesture database (`.dat`) file from the APK instead of relying on a Windows filesystem path.

**Comment by Byte247** [comment]
ah I think you are right the problem is in the database link. When building now I do get the "No gestures registered" error, which means the database was not loaded correctly. In the Documentation (for windows as I am debugging this on windows first) it 
wants me to add this :
```
#if UNITY_EDITOR
"gr.loadFromFile("myProject/myGestureDatabaseFile.dat");
#else
gr.loadFromFile(Application.streamingAssetsPath + "/myGestureDatabaseFile.dat");
#endif


```

but I do not have a gr object since I only use the MiVRy gesture recognition object as described in step 6 of the Documentation

**Comment by Byte247** [comment]
Here is my MiVRy Inspector: 
![MiVRy_inspector](https://github.com/MARUI-PlugIn/MiVRy/assets/119036910/0b0faa42-b05c-4a44-9f5f-f8c6792a5273)

This one works while in unity

**Comment by max-krichenbauer** [comment]
Do you have a Mivry license?
Personalized support is only given to people who support Mivry by purchasing a license.
Otherwise, since it's not a bug, it doesn't really belong in a github issue.

**Comment by Byte247** [comment]
hmm as we are only in the beginning of the project phase (for a student project) we do not have a license yet. Though I would argue it would belong into a bug/issue category, at least on the documentation side. We followed the documentation you sent, as well as this Youtube tutorial: https://youtu.be/EPlalwszVhA?si=g7WoZq9FtmXK10Ob and are still not able to use it in a build project. Others might learn to use your otherwise great package in a similar way but then get frustrated before considering buying the license for the final project as they as well can not build it.

In conclusion I believe a more clear explanation on how how step 6 and 7 in your documentation are related or how to build a project using only step 6 (if possible) would be beneficial for all partys.

**Comment by Byte247** [comment]
But I was able to solve it anyway. For some reason the StreamingAssets folder with the .dat files was not automatically moved into the build folder
